lsfg-vk: Add at v1.0.0

SlowNicoFish opened the pull request:
**Summary**

Add the lsfg-vk:package, which enables lossless scaling on linux

Resolves getsolus/packages#6122

**Test Plan**

Tested with vkcube according to lsfg githubs steps [here](https://github.com/PancakeTAS/lsfg-vk/wiki/Verifying-the-installation)


**Checklist**

- [x] Package was built and tested against unstable
- [ ] This change could gainfully be listed in the weekly sync notes once merged  <!-- Write an appropriate message in the Summary section, then add the "Topic: Sync Notes" label -->

malfisya commented:
Hi, thank you for your interest in contributing! We always welcome new contributors. While the PR itself looks fine, there are a few things to note:
- The usual procedure for adding new package to repository is by [submitting a package request first](https://help.getsol.us/docs/packaging/procedures/request-a-package), wait for the request to be accepted and then a PR can be filed. I don't see a request a yet for `lsfg-vk`, please make one
- We have a template for a pull request message which is specified [here](https://help.getsol.us/docs/packaging/creating-a-new-package#commit-message-format-for-new-packages). If you have [initialize git hooks](https://help.getsol.us/docs/packaging/prepare-for-packaging#initialize-git-hooks-for-linting), this template should automatically be applied.

I will draft this PR for now until the package request is filed. Feel free to ask us any question either here or on matrix.

SlowNicoFish commented:
> Hi, thank you for your interest in contributing! We always welcome new contributors. While the PR itself looks fine, there are a few things to note:
> 
> * The usual procedure for adding new package to repository is by [submitting a package request first](https://help.getsol.us/docs/packaging/procedures/request-a-package), wait for the request to be accepted and then a PR can be filed. I don't see a request a yet for `lsfg-vk`, please make one
> * We have a template for a pull request message which is specified [here](https://help.getsol.us/docs/packaging/creating-a-new-package#commit-message-format-for-new-packages). If you have [initialize git hooks](https://help.getsol.us/docs/packaging/prepare-for-packaging#initialize-git-hooks-for-linting), this template should automatically be applied.
> 
> I will draft this PR for now until the package request is filed. Feel free to ask us any question either here or on matrix.

Thank you fo the feedback. I have added the package request and adjusted the PR with the template you provided.

malfisya reviewed (dismissed):
Please add in description how you test this package, so we can reproduce it. [Example](https://github.com/getsolus/packages/pull/5995).

malfisya commented on packages/l/lsfg-vk/package.yml:
```diff
@@ -0,0 +1,41 @@
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=On
+
+    cd ui
```
I would prefer to using `pushd` and `popd`:
```
pushd ui
%cargo_build
popd
```

malfisya commented on packages/l/lsfg-vk/package.yml:
```diff
@@ -0,0 +1,41 @@
+build      : |
+    %ninja_build
+
+    cd ui
```
Same here too

malfisya commented on packages/l/lsfg-vk/package.yml:
```diff
@@ -0,0 +1,41 @@
+install    : |
+    %ninja_install
+
+    cd ui
```
Same here too

malfisya commented on packages/l/lsfg-vk/package.yml:
```diff
@@ -0,0 +1,41 @@
+release    : 1
+source     :
+    - git|https://github.com/PancakeTAS/lsfg-vk.git : v1.0.0
+homepage   : https://github.com/PancakeTAS/lsfg-vk/?tab=readme-ov-file
```
?tab=readme-ov-file => don't need this

malfisya commented on packages/l/lsfg-vk/package.yml:
```diff
@@ -0,0 +1,41 @@
+    %cargo_install lsfg-vk-ui
+    cd ..
+
+    install -Dm00644 $pkgfiles/lsfg-vk-ui.desktop -t $installdir/usr/share/applications
```
For a desktop/GUI app, it should at least include three component:
- desktop file
- icon file
- appstream metainfo

You don't need to include a desktop file to the tree,`lsfg-vk-ui` already has [desktop file and icon file](https://github.com/PancakeTAS/lsfg-vk/tree/develop/ui/rsc). You just have to [install it](https://github.com/PancakeTAS/lsfg-vk/wiki/Building-from-source#building-lsfg-vk-ui-from-source). For appstream metainfo, please read [this](https://help.getsol.us/docs/packaging/appstream-metainfo#writing-appstream-metainfo). This is all needed, so the package can be displayed in Gnome Software and Discover.

malfisya reviewed (changes requested):
The package works as it is but I need a few adjustments before merging it
